src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.csproj

```diff
@@ -21,8 +21,8 @@
     <PackageReference Include="MediatR" Version="11.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
     <PackageReference Include="SFA.DAS.Authorization.ProviderPermissions" Version="6.0.97" />
-    <PackageReference Include="SFA.DAS.CommitmentsV2.Api.Client" Version="100.0.6" />
-    <PackageReference Include="SFA.DAS.CommitmentsV2.Shared" Version="100.0.6" />
+    <PackageReference Include="SFA.DAS.CommitmentsV2.Api.Client" Version="100.0.17" />
```

Comment:
Is the package update required?

Reply:
It was because of using Commitments and the new Types and properties it introduced as part of the nuget. This can be rolled back once we've used APIM to return everything RPL related.